projects/angular-formio/package.json

```diff
@@ -31,7 +31,7 @@
     "@formio/js": "^5.0.0-rc.54",
     "zone.js": "~0.13.0 || ~0.14.0",
     "lodash": "^4.17.20",
-    "ngx-bootstrap": "^10.0.0 || ^11.0.0 || ^12.0.0",
+    "ngx-bootstrap": "^10.0.0 || ^11.0.0 || ^12.0.0 || ^18.0.0",
```

Comment:
Are versions 13 - 17 not supported here, or could we use the operator syntax, e.g. `"nxg-boostrap": "<= 18.x"`

Reply:
ngx-bootstrap just jumped straight from 12 to 18
https://www.npmjs.com/package/ngx-bootstrap?activeTab=versions